doc/reference/power_management/index.rst

```diff
@@ -480,9 +480,9 @@ the following configuration flags.
    This flag is enabled if the SOC interface and the devices support device power
    management.
 
-:code:`CONFIG_PM_DEVICE_IDLE`
+:code:`CONFIG_PM_RUNTIME`
```

Comment:
CONFIG_PM_DEVICE_IDLE is being removed in https://github.com/zephyrproject-rtos/zephyr/pull/33942
Does it make change to rename it here ?

Reply:
We agreed on not removing it, we can rename it to `CONFIG_PM_DEVICE_RUNTIME`, though. `IDLE` name is not clear.

Reply:
changed to `CONFIG_PM_DEVICE_RUNTIME`

Reply:
Will keep it and I am going to remove from that pr.